docs/_config.yml

```
@@ -2,7 +2,7 @@
 name: .NET client library for Digipost API
 
 # url of the download button
-downloadUrl: https://www.nuget.org/packages/digipost-api-client/
+downloadUrl: https://www.nugddet.org/packages?q=Digipost.Api.Client
```

Denne her ser ikke helt OK ut. 
```suggestion
downloadUrl: https://www.nuget.org/packages?q=Digipost.Api.Client
```